wolp 🐺

jvuong4 opened the pull request:
added all modfest wolves, with very slight tweaks to match more than a foxbox. these tweaks include:
- properly setting a particle texture in the models
- properly setting ambientocclusion to false in the models
- relevant namespace changes lol

please note that this does not add wolf recipes; i never expected for them to need a crafting recipe for the fest, so one will need to be added eventually :o

jvuong4 committed:
wolp 🐺

- added all modfest wolves, with very slight tweaks to match foxbox
- please note that this does not add wolf recipes; i never expected for them to need a crafting recipe for the fest, so one will need to be added eventually :o

Oliver-makes-code commented on src/client/resources/assets/more_than_a_foxbox/models/more_than_a_foxbox/wolf/box.json:
```diff
@@ -0,0 +1,196 @@
+				"up": {"uv": [1, 14, 0.5, 13], "texture": "#1"},
+				"down": {"uv": [1.5, 13, 1, 14], "texture": "#1"}
+			}
+		},
```
These will z-fight with the body

Oliver-makes-code reviewed (changes requested):
Just this one change, plus all the wolf variants models should use the regular wolf model as the base, like how the Snow Fox is based off of the Fox's model (https://github.com/Sweet-Berry-Collective/MoreThanAFoxbox/blob/fabric-1.21.8/src/client/resources/assets/more_than_a_foxbox/models/more_than_a_foxbox/snow_fox/box.json)

Oliver-makes-code commented:
I can handle recipes once this is merged inn

jvuong4 committed:
fixed z-fighting + wolf variants tweak

- legs are moved inwards so as to no longer have z-fighting with the body
- all wolf variants now use the pale wolf model as the base

Oliver-makes-code commented:
Will merge after ModFest ToyBox so people don't have to worry about migrating variants

jvuong4 commented:
sounds good! :D
